File: blog/server2/model/post.go
package model

import (
	"time"
	// "fmt"
	// "github.com/jinzhu/gorm"
)

type Post struct {
	ID        uint64    `gorm:"primary_key;auto_increment" json:"id"`
	Title     string    `gorm:"size:255;not null;unique" json:"title"`
	Content   string    `gorm:"size:255;not null;" json:"content"`
	AuthorID  int       `gorm:"not null" json:"author_id"`
	Author    User      `json:"author"`
	Like      int       `gorm: json:"like"`
	CreatedAt time.Time `gorm:"default:CURRENT_TIMESTAMP" json:"created_at"`
	UpdatedAt time.Time `gorm:"default:CURRENT_TIMESTAMP" json:"updated_at"`
}

func (Post *Post) TableName() string {
	return "post"
}

func (post *Post) Save(db *gorm.DB) {
	// fmt.Println()
	db.Create(&post)
}

File: blog/server3/model/post.js
const mongoose = require('mongoose');
const Schema = mongoose.Schema;

// Define our model
const postSchema = new Schema({
  title: String,
  content: String,  
  user:{type: Schema.Types.Mixed, ref: 'user'},
  time: Date,
  like:Number
});

// Create the model class
const ModelClass = mongoose.model('post', postSchema);
// Export the model
module.exports = ModelClass;